components/x-audio/src/redux/player-logic.js

```diff
@@ -91,7 +96,10 @@ export const middleware = (store, audio = new Audio()) => {
 	audio.addEventListener('waiting', () => store.dispatch(actions.loading()));
 	audio.addEventListener('stalled', () => store.dispatch(actions.loading()));
 	audio.addEventListener('loadstart', () => store.dispatch(actions.loading()));
-	audio.addEventListener('loadedmetadata', () => store.dispatch(actions.loading()));
+	audio.addEventListener('loadedmetadata', () => {
```

Comment:
Will the tracking be set up in time for autoplays?

i.e. `play()` is run straight away but then presumably `loadedmetadata` could fire after that? Or does it always need metadata before it can play? 

Soz if this makes no sense

Reply:
yeah I had a look at that, `loadedmetadata` is dispatched first. I haven't check what happens if the mp3 has been cached - ie the user _plays_, _closes_ and _plays_ the same . podcast again. Will check now